bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/io/RefreshingOAuth2CredentialsInterceptorTest.java

```diff
@@ -138,7 +139,9 @@ public Void call() throws Exception {
       }
     };
 
-    underTest = new RefreshingOAuth2CredentialsInterceptor(executorService, credentials);
+    underTest =
+        new RefreshingOAuth2CredentialsInterceptor(executorService, credentials,
+            new RetryOptions.Builder().build());
```

Comment:
Good point.  Backoff logic is tricky to test.  I'll spend an hour or two today trying to get a test in.